### PR TITLE
SVCS-2599 Add custom keys for payments service

### PR DESCRIPTION
**Pull request by AaronTrippaers**

### CODE REVIEW

**Review by chad-pelly — COMMENTED** [review]
is /appStore/recievedNotifications  missing, the SDK expects the response in camel case 🤔

**Comment by AaronTrippaers** [comment]
> is /appStore/recievedNotifications or in the SDK appStore.getNotifications missing? The SDK expects the response in camel case 🤔

That is not custom data, but data defined on our side I think. The SDK should keep converting it to camel case 😄

**Comment by chad-pelly** [comment]
> > is /appStore/recievedNotifications or in the SDK appStore.getNotifications missing? The SDK expects the response in camel case 🤔
> 
> That is not custom data, but data defined on our/apple's side I think. The SDK should keep converting it to camel case 😄

Yeah indeed that's the problem the type interfaces are in camel case 😂

File: src/services/payments/products.ts [hunk 1]
```diff
@@ -9,13 +9,20 @@ export default (
   httpAuth: HttpInstance
 ): PaymentsProductsService => ({
```
[the remark this answers is not in the document]
Comment (AaronTrippaers) [reply src/services/payments/products.ts:10]: That is only the request and since this is a node service, the update method is atm not affected 😄

**Review by wwaaijer-exh — APPROVED** [review]

**Pull request merged**